packages/code-du-travail-frontend/src/conventions/Search/getQueryType.js

```diff
@@ -0,0 +1,19 @@
+// detect query type (IA level++)
+const getQueryType = str => {
+  if (str.match(/^[\d .-]+$/) && str.length >= 2) {
+    if (str.length <= 4) {
+      return "idcc";
+    } else if (str.length === 9) {
+      return "siren";
```

Comment:
I don't see where you handle `siren` case ?

Reply:
not handled yet